### DISCUSSION TITLE
Re exporting a project with CLI

### DISCUSSION THREAD

**Discussion by AlexBeemer** [webstudio-is/webstudio-community#184] [Q&A]
Hey all! Looking for some help with the CLI locally. I've exported a Vanilla project from the CLI, and that seems to work. If I run npm run dev, everything seems ok. If I go back to webstudio and make changes however, after re building the project, none of the changes I've made are reflected in the site. I've tried relinking and also syncing, but it seems to not want to change. If I make an entirely new project, it works fine again the first time, but then when I update, nothing changes. Am I missing something?

**Comment by AlexBeemer** [comment]
I've also tried deleting the files and re building, still nothing.

**Comment by kof** [comment]
either use --preview  flag or publish before syncing
